src/src/ControllerQueue/C011_queue_element.cpp

```diff
@@ -6,6 +6,18 @@
 
 C011_queue_element::C011_queue_element() {}
 
+C011_queue_element::C011_queue_element(C011_queue_element&& other)
+  : uri(std::move(other.uri))
+  , HttpMethod(std::move(other.HttpMethod))
+  , header(std::move(other.header))
+  , postStr(std::move(other.postStr))
+  , idx(other.idx)
+  , _timestamp(other._timestamp)
+  , TaskIndex(other.TaskIndex)
+  , controller_idx(other.controller_idx)
+  , sensorType(other.sensorType)
+{}
+
```

Comment:
In general, *one should* prefer `=default` at least when ctor only deals with member copy and does not do anything in the body:
- no need to update ctor each time members are changed
- no way to forget members have changed
- noexcept for manually-typed ctros needs to be here to avoid copies with std containers which check for `std::is_nothrow_move_constructible<ContainedClass>`-> https://en.cppreference.com/w/cpp/types/is_move_constructible, but idk if this applies here)

Reply:
Sounds like a good idea, will have a look at it.
To be honest, the move operator itself is still quite new to me, so glad there is something like the =default (as it does like I think it does)